spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ExampleRequest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.spring;
+
+import javax.validation.constraints.NotNull;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Used as a example request object in {@link AnnotatedServiceRegistrationBean}
+ * and {@link GrpcServiceRegistrationBean}.
+ */
+public final class ExampleRequest {
+
+    /**
+     * Creates a new instance.
+     */
+    public static ExampleRequest of(@NotNull String serviceType,
+                                    @NotNull String methodName,
+                                    @NotNull Object exampleRequest) {
+        return new ExampleRequest(serviceType, methodName, exampleRequest);
+    }
+
+    private final String serviceType;
+    private final String methodName;
+    private final Object exampleRequest;
+
+    private ExampleRequest(String serviceType, String methodName, Object exampleRequest) {
+        this.serviceType = serviceType;
+        this.methodName = methodName;
+        this.exampleRequest = exampleRequest;
+    }
+
+    /**
+     * Return the service type of this {@link ExampleRequest}.
```

Comment:
`Return -> Returns`. 
Conventionally, Javadoc uses 3rd person present regarding verbs.
https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide

Reply:
Thank you for review. But, the code has been removed due to force push.